### DISCUSSION TITLE
How do I invite the bot?

### DISCUSSION THREAD

**Discussion by Tawko** [ToogaInc/ToogaBooga#207] [Support/Q&A] [answered]
Hi, I wanted to know how i could get Tooga-Booga on my server, everytime i try to go on to the discord invite link, it wont allow me.

**Comment by Deaththth** [comment]
Hi there
The bot is currently not finished as we have not yet added the verification to the live system.
Also the instance has not been moved to a host yet.
Until then we are deciding to keep it private as to first get into an ideal state and then allow other servers to add it.

**Comment by ewang2002** [comment, marked as answer]
HI,
I just saw this, but yes -- Tooga is currently not available to the public due to some crucial features being missing. Once those features are added, I will release them. This should ideally be done by mid-summer.
